data/spark.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Actually: This file needs to live in `data/apps` not in just `data`. Was wondering why it wasn't showing up.

Reply:
Ah. Again, my bad. I've fixed that.

Reply:
Cool it's showing up now: https://deploy-preview-22--app-talk.netlify.com/#spark Now only the image and then we're done here 👍

Reply:
I think I have the image in the right place now. Thanks for the help!

Reply:
Seems like you uploaded it to your 'master' branch but this Pull Request is based on your `patch-1` branch.